### PR TITLE
[TRANSLATION] Improve Simplified & Traditional Chinese translations

### PR DESCRIPTION
**Pull request by njlyf2011**
## Purpose

Improved Simplified and Traditional Chinese (including HK and TW) translations about SHELL32 and device configuration files. 

## Proposed changes

```
dll/win32/shell32/lang/zh-CN.rc
dll/win32/shell32/lang/zh-HK.rc
dll/win32/shell32/lang/zh-TW.rc
drivers/processor/processr/cpu.inf
drivers/storage/class/cdrom/cdrom.inf
drivers/storage/class/disk/disk.inf
media/inf/intl.inf
```

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-chinese

**Comment by njlyf2011** [comment]
My work at this stage has been completed, this pull request can be reviewed.

File: drivers/processor/processr/cpu.inf [hunk 1]
```diff
@@ -453,6 +453,66 @@ Processor.ServiceDesc = "Processor driver"
...
+AMDK8Processor.DeviceDesc = "AMD Athlon 64 / Turion 64 / Opteron / Sempron 處理器"
+AMDPHENProcessor.DeviceDesc = "AMD Athlon 64 X / Turion 64 X / Phenom 處理器"
+AMDQProcessor.DeviceDesc = "AMD Athlon 64 Q* / Turion 64 *M / Sempron S* 處理器"
+TransmetaProcessor.DeviceDesc = "Transmeta Crusoe 處理器"
+Standard = "標準處理器"
+ViaC7Processor.DeviceDesc = "VIA C7 處理器"
+ViaNEHProcessor.DeviceDesc = "VIA Nehemiah 處理器"
+ViaESTProcessor.DeviceDesc = "VIA Esther 處理器"
+ViaNANOProcessor.DeviceDesc = "VIA Nano 處理器"
+ViaProcessor.DeviceDesc = "VIA 處理器"
```
[the remark this answers is not in the document]
Comment (njlyf2011) [reply drivers/processor/processr/cpu.inf:514]: > 

I think it's easier to maintain two traditional Chinese-based languages next to each other.

[the remark this answers is not in the document]
Comment (njlyf2011) [reply drivers/processor/processr/cpu.inf:514]: I'm about to revise to the regular style.
我还是改回常规样式吧，毕竟这样不符合规范。

**Review by eason329 — APPROVED** [review]
OK LGTM now.

**Comment by kernelbin** [comment]
I glanced at the PR, LGTM so far. If I've got more free time I will review again, I will approve for now!

File: drivers/processor/processr/cpu.inf [hunk 2]
```diff
@@ -482,3 +512,35 @@ ViaNEHProcessor.DeviceDesc = "VIA Nehemiah 处理器"
 ViaESTProcessor.DeviceDesc = "VIA Esther 处理器"
 ViaNANOProcessor.DeviceDesc = "VIA Nano 处理器"
 ViaProcessor.DeviceDesc = "VIA 处理器"
+
```
Comment (binarymaster) [inline drivers/processor/processr/cpu.inf:515]:
```suggestion
```

**Review by binarymaster — APPROVED** [review]

**Pull request merged**